From 7f4c84a75e6df348f9a4365d5fa18b30ecacdbf8 Mon Sep 17 00:00:00 2001
From: Sanyam Asthana <sanyam.asthana@gmail.com>
Date: Sat, 21 Mar 2026 22:26:12 +0530
Subject: [PATCH 08/10] make font loading faster

---
 index.html | 2 ++
 styles.css | 1 +
 2 files changed, 3 insertions(+)

diff --git a/index.html b/index.html
index dc04991..954cddc 100644
--- a/index.html
+++ b/index.html
@@ -7,6 +7,8 @@
         <meta name="description" content="Sanyam Asthana's Portfolio">
         <meta name="author" content="Sanyam Asthana">
         <meta name="keywords" content="Sanyam Asthana, IIIT Hyderabad, computational linguistics, natural language processing, systems programming, cybersecurity, programming languages">
+
+        <link rel="preload" href="JetBrainsMonoNerdFont-Regular.ttf" as="font" type="font/ttf" crossorigin>
     </head>
     <body>
 
diff --git a/styles.css b/styles.css
index 75165d1..86238cb 100644
--- a/styles.css
+++ b/styles.css
@@ -28,6 +28,7 @@
     src: url('JetBrainsMonoNerdFont-Regular.ttf') format('truetype');
     font-weight: normal;
     font-style: normal;
+    font-display: swap; /* Swap the font as soon as it is loaded */
 }
 
 body {
-- 
2.50.1 (Apple Git-155)

